Source/panels/spell_book.cpp

```diff
@@ -184,16 +183,16 @@ void DrawSpellBook(const Surface &out)
 			} break;
 			}
 		}
-		yp += SpellBookDescriptionHeight;
+		yp += SpellBookDescription.height;
 	}
 }
 
 void CheckSBook()
 {
-	Rectangle iconArea = { GetPanelPosition(UiPanels::Spell, { 11, 18 }), { 48 - 11, 314 - 18 } };
-	Rectangle tabArea = { GetPanelPosition(UiPanels::Spell, { 7, 320 }), { 311 - 7, 349 - 320 } };
+	// Icons are drawn in a column near the left side of the panel and aligned with the spell book description entries
+	Rectangle iconArea = { GetPanelPosition(UiPanels::Spell, { 11, 18 }), Size { 37, SpellBookDescription.height * 7 } };

```

Comment:
I think this is intended (to make hit size of all buttons the same), but just to make sure:
The height is different between master and this pr
master: 314 - 18 = 296
pr: 43 x 7 = 301

Reply:
Looking at this more I think the position of the hit area for spell book icons is also wrong, it should start at 11,12 instead of 11,18 to match how they're drawn. So the bottom edge was maybe correct previously but the top was off, in the current state of this PR the hit area has grown off the bottom when it should've grown from the top. I'll fix that now.

Reply:
Ok, that turned out to be annoying but I'll leave it for the future.

![image](https://user-images.githubusercontent.com/357684/174477190-273a6ed5-eb53-4d24-ac42-2d2136b6b898.png)
This is the hit area for a spell icon button (in vanilla Diablo/Hellfire and devilutionX). The behaviour is (now) not changed by this PR, but the margin between one button and the next is included in the hit area for the topmost button. It should really be a set of 37x38 pixel hit regions.

Reply:
I also noticed that the selection behavior in master is not 100% pixel correct, but that's a really minor issue.
That's why I thought the extra 5 pixel don't make a big difference anyway.
Thanks for looking again. 🙂